src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -801,6 +806,7 @@ configure(struct sched_cmd *cmd)
 				has_acc_cu = 1;
 		}
 
+		control = cfg->data[i] & 0x7; //bit [2:0]
```

Comment:
It is a bit confusing that we are using two different mask here. ~ZOCL_KDE_MASK and 0x7. Can we unify it? Something like
control = control = cfg->data[i] & 0x7; //bit [2:0]
if (zocl_xclbin_accel_adapter(control))
?

Reply:
actually they are 2 different mask, the control information is in lower 3 bits and zocl_kde_mask is 8 bit.

Reply:
I do believe the ZOCL_KDS_MASK actually mean bit [2:0]. If bits [7:3] (max ctx) are ever used, kds_mask == ACCEL_ADAPTER in zocl_xclbin_accel_adapter would be wrong. 
But I am not going to fight for it but just leave a comment. :)